test/e2e/end_user_test.go

```diff
@@ -51,4 +85,65 @@ var _ = Describe("Openshift on Azure end user e2e tests [EndUser]", func() {
 		// Expect(kerrors.IsForbidden(err)).To(Equal(true))
 		fmt.Printf("PDB create error: %v\n", err)
 	})
+
+	It("should deploy a template and check the visit counter increments", func() {
+		var regex = regexp.MustCompile(`\<span.*id=\"count-value\"\>(?P<counter>\d+)\<\/span\>`)
+
+		// Create the template
+		template, err := c.tc.Templates("openshift").Get(
+			"nodejs-mongodb-example", metav1.GetOptions{})
+		Expect(err).NotTo(HaveOccurred())
+
+		// Create a secret to hold the template params
+		secret, err := c.cc.Secrets(c.namespace).Create(&corev1.Secret{
```

Comment:
the secret shouldn't be necessary and I don't think using it adds anything to the smoke test